src/workloads/query/multiplanner/MultikeyIndexes.yml

```diff
@@ -412,5 +412,7 @@ AutoRun:
           - standalone-80-feature-flags # At time of writing this will enable PM-3591.
           - standalone-all-feature-flags # At time of writing this will enable PM-3591.
           - standalone-classic-query-engine
+          - replica
```

Comment:
Same case - I think we could remove the non-feature-flag replica case here as there is no standard standalone case running